about.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WTA Summit</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <main class="about" id="landingPage">
+      <div class="show-background">&nbsp;</div>
+      <header class="nav-header">
+        <nav>
+          <!-- top nav start -->
+          <div class="top">
+            <svg class="top__nav-icon">
+              <use
+                xlink:href="./assets/icons/symbol-defs.svg#icon-social-facebook"
+              ></use>
+            </svg>
+            <svg class="top__nav-icon">
+              <use
+                xlink:href="./assets/icons/symbol-defs.svg#icon-social-twitter"
+              ></use>
+            </svg>
+            <a href="#" class="top__nav-link"> English </a>
+            <a href="#" class="top__nav-link"> My Page </a>
+            <a href="#" class="top__nav-link">Logout</a>
+          </div>
+          <!-- Top Nav end -->
+
+          <!-- nav start -->
+          <div class="nav">
+            <svg class="nav__menu-icon">
+              <use xlink:href="./assets/icons/symbol-defs.svg#icon-menu"></use>
+            </svg>
+            <div class="closeicon-wrapper">
+              <div class="nav__closeicon-wrapper">
+                <a href="./index.html" class="nav__logo-small">
+                  <img
+                    src="./assets/img/Women-in-Tech-Logo.png"
+                    alt="Women in tech logo"
+                  />
+                </a>
+                <svg class="nav__closeicon">
+                  <use xlink:href="./assets/icons/symbol-defs.svg#icon-clear"></use>
+                </svg>
+              </div>
+            </div>
+            <div class="nav__menu">
+              <a href="./index.html" class="nav__logo">
+                <img
+                  src="./assets/img/Women-in-Tech-Logo.png"
+                  alt="Women in tech logo"
+                />
+              </a>
+              <div class="nav__items">
+                <ul class="nav__list">
+                  <li class="nav__list-items">
+                    <a href="./about.html" class="nav__list-link">About</a>
```

Comment:
- [ ] Kindly change this link from about.html to index.html to users can easily navigate back to the home page from the about page 👍🏿